tools/nni_cmd/config_schema.py

```diff
@@ -36,6 +36,7 @@
 Optional('logDir'): os.path.isdir,
 Optional('debug'): bool,
 Optional('logLevel'): Or('trace', 'debug', 'info', 'warning', 'error', 'fatal'),
+Optional('remoteLoggingType'): Or('http', 'none'),
```

Comment:
Suggest not to put this on top level,  maybe under trial or traininig service , and rename it to loggingType or a better name.

Reply:
As discussed offline, we will change this param to `logCollection`